airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -320,7 +354,7 @@ protected static Properties getDebeziumProperties(JsonNode config, ConfiguredAir
     props.setProperty("plugin.name", "pgoutput");
     props.setProperty("connector.class", "io.debezium.connector.postgresql.PostgresConnector");
     props.setProperty("offset.storage", "org.apache.kafka.connect.storage.FileOffsetBackingStore");
-    props.setProperty("offset.storage.file.filename", "/tmp/offsets-" + RandomStringUtils.randomAlphabetic(5) + ".dat");
+    props.setProperty("offset.storage.file.filename", AirbyteFileOffsetBackingStore.DEFAULT_OFFSET_STORAGE_PATH.toString());
```

Comment:
This shouldn't be a constant ultimately, but likely outside of the scope of this PR?

Reply:
yeah. i'll double check that i can't easily fix it here. it relates to what i was saying about the reading / writing the debezium offset file so i might be able to fix that as i address your concern there.

Reply:
fixed it.